### DISCUSSION TITLE
Should Debian package 1.14 RC1 when it freezes in January 2023

### DISCUSSION THREAD

**Discussion by paulgevers** [lwindolf/liferea#1158] [Q&A] [answered]
@lwindolf I just saw your announcement of liferea 1.14 RC1. Normally I'm quick to package the latest version of liferea for Debian, but the first [freeze](https://release.debian.org/testing/freeze_policy.html) for `bookworm` is happening on 2023-01-12 and I shouldn't plan to upload non-bug fix releases after 2023-02-05. Should I aim to have 1.14 in `bookworm` or is it better to stay with 1.13?

**Comment by lwindolf** [comment, marked as answer]
@paulgevers My vague timeline is to have 1.14 before Xmas 2022. So I believe it would be safe to use 1.14

**Comment by paulgevers** [comment]
Thank you.
